.github/PULL_REQUEST_TEMPLATE/release_pull_request_template.md

```diff
@@ -0,0 +1,55 @@
+## Description
+
+This PR introduces a new <!-- insert new version here --> release in tezos-packaging.
+
+<!-- Depending on the type of the release some of the points below can be ommited.
+
+E.g. in case we introduce a new changes to our native packages, it's not necessary to bump
+used Tezos sources, create a new release in this repository and update brew formulas.
+
+Feel free to cross out points that aren't needed in some specific case.
```

Comment:
I think we shouldn't do this, if need to make e.g. a second PR after the opam packages are updated it would be great to be able to look at the previous one and see which points where missing IMO

Reply:
In fact, I think here we should add something like `Resolves `, but instead since it might relate to a previous PR we should have `Follows #`